files/zh-cn/web/progressive_web_apps/tutorials/cycletracker/index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: 周期追踪器
+slug: Web/Progressive_web_apps/Tutorials/CycleTracker
+---
+
+{{NextMenu("Web/Progressive_web_apps/Tutorials/CycleTracker/HTML_and_CSS")}}
+
+{{PWASidebar}}
+
+这个入门级教程将带领你完成构建基础渐进式 Web 应用（PWA）的所有步骤。我们将使用 Web 技术——HTML，CSS 和 JavaScript——来构建一个称为“周期追踪器”的月经周期跟踪 Web 应用。像所有的 Web 应用一样，周期追踪器设计成在所有设备的所有浏览器上工作。
+
+我们将逐步构建一个功能完整的 Web 应用，然后逐步改善周期追踪器，使其可安装并在用户离线时仍可工作。
+
+默认情况下，PWA 是常规网站，使用相同的技术构建。就像常规网站一样，PWA 是可链接的，可通过搜索引擎发现，并在浏览器中可见。通过包含一个清单文件和 service worker，并通过 SSL 提供网站，任何网站都可以成为 PWA。
+
+## PWA 的优势
+
+使用 Web 语言，我们将创建一个完全功能的应用，该应用在线和离线都能工作，同时在浏览器和用户的操作系统（OS）上都能运行。像任何常规网站一样，周期追踪器托管在 Web 服务器上并可从 Web 服务器下载。我们需要的只是一个文本编辑器：周期追踪器，像所有的 PWA 一样，不需要任何额外的编程语言知识、打包或专有的 SDK。周期追踪器，像任何 PWA 一样，可以无缝地安装在任何操作系统上，而无需应用商店（也不需要应用商店的批准和费用）。
+
+- 使用标准和开放的 Web 技术
+
+  - ：历史上，为了使应用程序可在操作系统（如 Windows、iOS、MacOS、Linux 和 Android）上安装，应用程序是用操作系统支持的编程语言（如 C#、.Net、Objective C、Swift、Kotlin、Java 或 Python）开发的。PWA 基于一个不同的模型：它们使用单一的代码库，使用标准的开放 Web 技术（HTML、CSS 和 JavaScript）编写，可以在各种操作系统上运行。
+
+- 无需编译
+
+  - ：对于大多数编程语言——如 Java、C/C++ 和 Kotlin，这些常用于构建 Android 应用，以及 Objective-C 和 Swift，用于 iOS——需要将代码编译和打包成可安装的格式，如 exe、.dmg、.elf 和 .apk 或其他依赖于操作系统的可安装文件类型。根据语言，编译和打包可能需要操作系统的 {{glossary("SDK")}} 。PWA 使用每个操作系统都支持的 Web 技术，不需要打包或编译。是的，开发团队可以有复杂的构建系统，但是，正如我们在构建周期追踪器时将展示的，PWA 可以仅通过 HTML 和 JavaScript 构建（以及 CSS，尽管样式不一定是 PWA 所必需的）。
+
+- 可以在任何地方和任何时候使用
+
+  - ：单一操作系统的应用程序通过下载向用户分发，通常在专有的应用商店中。他们通过像苹果 App Store、[Google Play](https://play.google.com/store/apps)、[Microsoft Store](https://apps.microsoft.com/store/apps) 或类似的供应商提供。PWA 没有这些要求。如果你想分发你的周期追踪器应用，你不需要一个中间人。用户可以通过访问其在线版本来访问你的应用。但是如果你想要的话，你可以在 Play Store 和 App Store 上分发你的 PWA，与其他 iOS 或 Android 应用无任何区别。
+
+- 用户易于安装
+
+  - ：历史上，下载的单一操作系统应用程序必须由用户有意识地安装。根据操作系统、安装格式和下载源，这可能是一个多步骤的安装过程。PWA 更为简洁。PWA 对任何支持的浏览器都可用，并且可以通过几次点击[安装](/zh-CN/docs/Web/Progressive_web_apps/Guides/Installing)。
```

Comment:
Please fix all the definition list, we have documentation how to write this [here](https://developer.mozilla.org/zh-CN/docs/MDN/Writing_guidelines/Howto/Markdown_in_MDN#%E5%AE%9A%E4%B9%89%E5%88%97%E8%A1%A8).